.github/workflows/comment-issue.yml

```
@@ -0,0 +1,45 @@
+name: Comment Issue
+
+on:
+  issues:
+    types:
+      - labeled
+
+jobs:
+  add-comment-for-user-interest:
+    if: "github.event.label.name == 's: waiting for user interest'"
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Add Comment For User Interest
+        uses: actions/github-script@v6
+        with:
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `Thanks for feature proposal.
+
+                We marked it as "waiting for user interest" for now to gather some feedback from our community:
+
+                - If you would like to see this feature being implemented, please react to the description with an up-vote (:+1:).
+                - If you have a suggestion or want to point out some special cases that needs to be considered, please leave a comment, so we are aware on them.
+
+                We would also like to hear about your use cases for the feature to give us a better understanding of potentially implied or explicit requirements.
```

Shouldn't that already be included in the issue description?

---

Yes, but only for the person asking for the feature, not others who might want to use it.

```suggestion
                We would also like to hear about other community member's use cases for the feature to give us a better understanding of potentially implied or explicit requirements.
```

---

members' not member's